src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -189,21 +189,21 @@ internal IEnumerable<AttachmentNameWithCount> GetAttachmentsMetadataForDocumentW
             }
         }
 
-        public void RetireAttachment(DocumentsOperationContext context, AttachmentDetailsServer attachment, Slice keySlice)
+        public void RemoteAttachment(DocumentsOperationContext context, AttachmentDetailsServer attachment, Slice keySlice)
```

Comment:
this is not a verb any more so it would be great if you could adjust the method name